offload/DeviceRTL/include/Interface.h

```diff
@@ -359,6 +359,10 @@ int32_t __kmpc_cancel(IdentTy *Loc, int32_t TId, int32_t CancelVal);
 ///{
 int32_t __kmpc_shuffle_int32(int32_t val, int16_t delta, int16_t size);
 int64_t __kmpc_shuffle_int64(int64_t val, int16_t delta, int16_t size);
+
+/// Manage dynamic schedule data structure:
+void __init_thread_dst();
```

Comment:
This function is internal so no need to make it a C interface, as well as with `__` prefix. Technically better to wrap it into some namespace.